front/public/locales/fr/rollingstock.json

```
@@ -71,7 +71,8 @@
     "rollingStockDuplicateName": "Un matériel roulant du même nom existe déjà.",
     "rollingStockNotAdded": "Le matériel roulant n'a pas été ajouté.",
     "rollingStockNotUpdated": "Le matériel roulant n'a pas été mis à jour.",
-    "rollingStockNotDeleted": "Le matériel roulant n'a pas été supprimé."
+    "rollingStockNotDeleted": "Le matériel roulant n'a pas été supprimé.",
+    "invalidEffortCurves": "Les valeurs de vitesse et d'effort doivent être strictement positives, et les valeurs de vitesse doivent être uniques (sans doublons)"
```

See comment in utils

```suggestion
    "invalidEffortCurves": "Les valeurs de vitesse doivent être uniques (sans doublons)"
```